add input group in system gid range

as expected by the postinst of Debian udev package.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Abdur Rehman <abdur.rehman@toradex.com>

---
 group.master |    1 +
 1 file changed, 1 insertion(+)

Index: base-passwd-3.5.26/group.master
===================================================================
--- base-passwd-3.5.26.orig/group.master
+++ base-passwd-3.5.26/group.master
@@ -12,6 +12,7 @@ uucp:*:10:
 man:*:12:
 proxy:*:13:
 kmem:*:15:
+input:*:101:
 dialout:*:20:
 fax:*:21:
 voice:*:22:
